docs/reference/contributing/guidelines/workflow.md

```diff
@@ -194,7 +204,7 @@ We use many other labels to summarize the scope and effect of the changes.
 - *needs: preceding PR* - This pull request cannot yet be merged because it has a dependency on another pull request that needs to merge first.
 - *DO NOT MERGE* - This pull request contains changes that may be in a draft state and submitted purely for review, or may contain breaking changes that have not been considered.
 - *devices: 'name'* - The pull request specifically affects the named device(s).
-- *component: 'name'* - The pull request specifically affects the named component.
+- *component: 'name'* - The pull request specifically affects the named component. Component names follow the structure of Mbed OS (`ble`, `storage`, `tls` and so on).
```

Comment:
Query: When we say component here, do we mean the new components, such as the new storage ones?

Reply:
All the components we have (Mbed OS directory structure to be specific) such as ble, storage, drivers, hal , etc

Reply:
@sg- Any suggestions?